src/engine/Server.cpp

```diff
@@ -464,7 +475,8 @@ ad_utility::websocket::OwningQueryId Server::getQueryId(
 auto Server::cancelAfterDeadline(
     const net::any_io_executor& executor,
     std::weak_ptr<ad_utility::CancellationHandle> cancellationHandle,
-    std::chrono::seconds timeLimit) {
+    std::chrono::seconds timeLimit)
```

Comment:
Write at one central place (probably in the Constants.h, s.t. also the parameters see it a 
`using TimeLimit = std::chrono::seconds` and then consistently use this alias. 
That way we can easily replace it in the future (should use such aliases in general).